EIPS/eip-3475.md

```diff
@@ -388,38 +384,37 @@ The `_amount` is the amount of the bond for which the spender is approved.
 
 However, any existing [EIP-20](./eip-20.md) token contract can issue its bonds by delegating the minting role to a bank contract with the interface of this standard built-in. Check out our reference implementation for the correct interface definition.
 
-To ensure the reading of transactions, "Issue", "Redeem", "Burn" and "Transfer", Events cited above MUST be emitted when such transaction is passed.
+To ensure the reading of transactions, "Issue", "Redeem", "Burn" and "Transfer", events cited above MUST be emitted when such transaction is passed.
 
-**Note that the this standard interface is also compatible with [EIP-20](./eip-20.md) and [EIP-721](./eip-721.md) interface.**
+**Note that the this standard interface is also compatible with [EIP-20](./eip-20.md) and [EIP-721](./eip-721.md) and [EIP-1155](./eip-1155.md)interface.**
 
 However, creating a separate bank contract is recommended for reading the bonds and future upgrade needs.
 
 Acceptable collateral can be in the form of [EIP-20](./eip-20.md) tokens, [EIP-721](./eip-721.md) tokens, or other bonds represented by the standard.Thus bonds can now represent a collection of collaterals (of the same type) of all fungible/non-fungible or semi-fungible tokens.
 
 ## Test Cases
 
-Testcase for the minimal reference implementation is available [here](../assets/eip-3475/ERC3475.test.ts). Developers need to run this script with truffle dependency in the npm package.
+Test-case for the minimal reference implementation is available [here](../assets/eip-3475/ERC3475.test.ts). Use the Truffle box to compile and test the contracts.
 
 ## Reference Implementation
 
-[Reference Interface](../assets/eip-3475/IERC3475.sol).
-  - This demonstration shows only minimalist implementation.
+- [Interface](../assets/eip-3475/interfaces/IERC3475.sol).
 
-[Debond-EIP-3475](../assets/eip-3475/Debond-ERC3475.sol) 
-  - Real use case consisting of more sophisticated logic like:
-    - Different classes of bonds (fixed/floating rate bonds).
-    - Redemption conditions.
+- [Basic Example](../assets/eip-3475/ERC3475.sol).
+      - This demonstration shows only minimalist implementation.
 
 ## Security Considerations
 
 - The `function setApprovalFor()` gives the operator role in this standard. It has all the permissions to transfer, burn and redeem bonds by default.
-- If the owner wants to give a one-time allocation to an address, he should call the `function approve()` giving the `Transaction[]` allocated. 
-- The `function issue()` can only be called by the bank contract (what we call issuer in this EIP).
+
+- If the owner wants to give a one-time allocation to an address for specific bonds(classId,bondsId), he should call the `function approve()` giving the `Transaction[]` allocated rather than approving all the classes using `setApprovalFor`.
```

Comment:
```suggestion
- If the owner wants to give a one-time allocation to an address for specific `bonds(classId,bondsId)`, he should call the `function approve()` giving the `Transaction[]` allocated rather than approving all the classes using `setApprovalFor`.
```